src/openrct2/ride/Vehicle.cpp

```diff
@@ -4823,14 +4823,14 @@ void Vehicle::UpdateRotating()
         timeToSpriteMap = Rotation3TimeToSpriteMaps[sub_state];
     }
 
-    int32_t time = current_time;
+    uint8_t time = current_time;
```

Comment:
Will changing to `uint32_t` not work?

Reply:
This change makes very little sense and it will definitely break the lookup, current_time is int16_t, there is more questionable code below that, also signed integer overflow is UB in C++, so there is also that. I think it would be best to turn current_time to uint16_t and from there actually stick to that type.